tools/developer.html

```diff
@@ -1,22 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <title>highlight.js developer</title>
 
   <link rel="stylesheet" href="../src/styles/default.css">
 
   <style>
-    body { padding: 20px;
-    font-family: sans-serif;
-     }
+    body {
```

Comment:
Please undo all the linting type changes to this file and only include the 2-3 line changes that are relevant to this PR.

Reply:
undid all of my linting changes, sry